src/docs/Ways of Working/git.md

```diff
@@ -0,0 +1,105 @@
+---
+sidebar_position: 4
+slug: /git-flow
```

Comment:
I'd recommend `ways-of-working/git-flow` here because it's under a drop down. Technically erverything is at the same level but better to mark sections in url too.